docs/openapi.yaml

```diff
@@ -213,6 +213,72 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Publication'
+  /{publicationIdentifier}/publish:
```

Comment:
Here we're doing HTTP-RPC; it is worth considering if this is the pattern we want. If it is, we need to document that we are doing /{noun}/{id}/{verb} and that this has a particular meaning. I'd be tempted to differentiate the verb further with some marker (such as Google's :verb)

Reply:
Alternative is to place this under `PUT: /publication/{publicationIdentifier}` with customer request body. But I think that update handler does already a lot of things, but thats probably ok?

Reply:
Alternative is to place this under `PUT: /publication/{publicationIdentifier}` with custom request body. But I think that update handler does already a lot of things, but thats probably ok?